### DISCUSSION TITLE
update_many for metrics

### DISCUSSION THREAD

**Discussion by kulbachcedric** [online-ml/river#995] [Ideas]
Hi, 

we could add a ```update_many(y,y_pred)``` method for the metrics.
It could be helpful with the ```predict_proba_many``` and ```learn_many``` methods.

What do you think?

Cedric

**Comment by MaxHalford** [comment]
Yes that's a reasonable ask. I'll think about it! It might have a few implications. In the meantime I'll add a card to the roadmap.
